### PR TITLE
[ISSUE #4420] Add Feishu Sink connector

### LINKED ISSUE #4420: [Feature] Add Feishu sink connector
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Feature Request

Add Feishu sink connector

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [ ] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct) *

### PR DESCRIPTION
**Pull request by SunnyBoy-WYH**
Fixes #4420.

### Motivation

[ISSUE #4420] Add Feishu Sink connector

### Modifications

add sink connector for feishu：

- support open_id/chatId  etc
- feishu doc:https://open.feishu.cn/document/server-docs/im-v1/message/create?appId=cli_a5a556ae023a100d
- like promthaus connector, i import guava-retry tool and keep 2.0.0 version
image

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by xwm1992** [comment]
@SunnyBoy-WYH ping, please take a look for the reveiw and fix the ci check error, thanks.

**Comment by SunnyBoy-WYH** [comment]
> @SunnyBoy-WYH ping, please take a look for the reveiw and fix the ci check error, thanks.

ok，i will solve it later. thanks

**Comment by SunnyBoy-WYH** [comment]
> It seems that you haven't fixed the previous CI error.

please approve the ci pipeline.

**Comment by SunnyBoy-WYH** [comment]
> CI errors:
> 
> ```
> [ant:checkstyle] [WARN] /home/runner/work/eventmesh/eventmesh/eventmesh-connectors/eventmesh-connector-feishu/src/main/java/org/apache/eventmesh/connector/feishu/sink/connector/FeishuSinkConnector.java:105:108: '.' should be on a new line. [SeparatorWrapDot]
> 
> [ant:checkstyle] [WARN] /home/runner/work/eventmesh/eventmesh/eventmesh-connectors/eventmesh-connector-feishu/src/main/java/org/apache/eventmesh/connector/feishu/sink/connector/FeishuSinkConnector.java:106:51: ',' is not followed by whitespace. [WhitespaceAfter]
> [ant:checkstyle] [WARN] /home/runner/work/eventmesh/eventmesh/eventmesh-connectors/eventmesh-connector-feishu/src/main/java/org/apache/eventmesh/connector/feishu/sink/connector/FeishuSinkConnector.java:141:63: WhitespaceAround: '{' is not preceded with whitespace. [WhitespaceAround]
> [ant:checkstyle] [WARN] /home/runner/work/eventmesh/eventmesh/eventmesh-connectors/eventmesh-connector-feishu/src/main/java/org/apache/eventmesh/connector/feishu/sink/config/FeishuSinkConfig.java:19:1: 'import' should be separated from previous line. [EmptyLineSeparator]
> > Task :eventmesh-connectors:eventmesh-connector-feishu:checkstyleMain FAILED
> ```
> 
> You can refer to https://eventmesh.apache.org/community/contribute/contribute#code-style to help you perform a checkstyle check locally.

Done

**Review by Pil0tXia — APPROVED** [review]

**Review by xwm1992 — APPROVED** [review]

File: eventmesh-connectors/eventmesh-connector-feishu/src/main/resources/sink-config.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  idc: FT
+  env: PRD
+  group: feishuSink
+  appId: 5031
+  userName: feishuSinkUser
+  passWord: feishuPassWord
+connectorConfig:
+  connectorName: feishuSink
+  reciveId: reciveIdValue
+  reciveType: open_id
```
Comment (hhuang1231) [inline eventmesh-connectors/eventmesh-connector-feishu/src/main/resources/sink-config.yml:30]: The `reciveId` and `recoverType` fields in the `sink-config.yml` are different from the field names in `SinkConnectorConfig`, which will cause the configuration to fail to be imported correctly.

@SunnyBoy-WYH 配置文件中reciveId和reciveType字段与SinkConnectorConfig中字段名不同，这将导致配置无法正确导入。

Comment (hhuang1231) [reply eventmesh-connectors/eventmesh-connector-feishu/src/main/resources/sink-config.yml:30]: @pandaapo ok, I'll try to perfect it and make it work.

**Pull request merged**